sdk/resourcemanager/communication/armcommunication/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 2.0.0-beta.2 (2022-08-14)
+### Other Changes
+- Replace API version `2021-10-01-preview` with `2022-07-01-preview`
```

Comment:
This is a preview swagger introducing a lot of breaking changes.  We should see what we can do to keep this as a v1.1.0-beta.1 release

Reply:
We have already released the 2.0.0-beta.1 in June. This release is requested from service team and it seems only api version changed. For any preview version with breaking changes, we usually pump major version and add beta suffix. If we need to retreat 2.0.0-beta.1 and make the release a minor version, we need to handle all the breakings in 2.0.0-beta.1.  But in my opinion, it is hard and weird to do so.